src/components/paneler/paneler.less

```diff
@@ -89,6 +88,10 @@
 }
 
 @media screen and (min-width: 641px) {
+    .ekspanderbartPanel__innhold {
+        padding: 0 !important;
+    }
```

Comment:
Har foreløpig ikke funnet ut hvorfor denne må `!important` for å slå til.

Reply:
Paddingen settes her https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-ekspanderbartpanel-style/src/index.less#L52 Hvorfor ønsker vi å overstyre stylingen fra nav-frontend?